packages/dev/core/src/DeviceInput/InputDevices/webDeviceInputSystem.ts

```diff
@@ -592,13 +592,11 @@ export class WebDeviceInputSystem implements IDeviceInputSystem {
         const noop = function () {};
 
         try {
-            const options: object = {
-                passive: {
-                    get: function () {
-                        passiveSupported = true;
-                    },
+            const options = Object.defineProperty({}, "passive", {

```

Comment:
Why changing it to this ? just as an FYI for me :-)

Reply:
When I tested the original code, passiveSupported was always coming back as false.  I had found this [documentation](https://github.com/WICG/EventListenerOptions/blob/gh-pages/explainer.md#feature-detection) that used Object.defineProperty and when I tested that, it evaluated properly so I subbed it in.